### PR TITLE
ref(dx): Improve package.json usages

### PR DESCRIPTION
**Pull request by anonrig**
Using `yarn` inside a package.json scripts value causes `200ms` increase by booting yarn twice.

### CODE REVIEW

**Review by scttcper — APPROVED** [review]

**Review by evanpurkhiser — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -220,11 +220,11 @@
   "proxyURL": "http://localhost:8000",
   "scripts": {
     "test-wrap": "node scripts/test.js",
```
Comment (evanpurkhiser) [inline package.json:222]: Can we kill this then?

Comment (anonrig) [reply package.json:222]: If no-one uses it, yes. Maybe some engineer does?

Comment (evanpurkhiser) [reply package.json:222]: almost for sure not

Comment (anonrig) [reply package.json]: removed it in a new commit @evanpurkhiser

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**